src/Microsoft.DotNet.Interactive.HttpRequest.Tests/ParserTests.Lexer.cs

```diff
@@ -20,11 +20,11 @@ public void multiple_whitespaces_are_treated_as_a_single_token()
 
             result.SyntaxTree.RootNode
                   .ChildNodes.Should().ContainSingle<HttpRequestNode>().Which
-                  .MethodNode.ChildTokens.First().Should().BeOfType<HttpSyntaxToken>();
+                  .ChildTokens.First().Should().BeOfType<HttpSyntaxToken>();
 
             result.SyntaxTree.RootNode
                   .ChildNodes.Should().ContainSingle<HttpRequestNode>().Which
-                  .MethodNode.ChildTokens.Single().Text.Should().Be("  \t  ");
+                  .ChildTokens.Single().Text.Should().Be("  \t  ");
```

Comment:
Consider pulling out a shared variable for `.ChildTokens.Single()` and use that in both assertions.